src/fidesops/task/graph_task.py

```diff
@@ -158,22 +200,43 @@ def pre_process_input_data(self, *data: List[Row]) -> NodeInput:
             )
 
         output: Dict[str, List[Any]] = {}
+
+        (
+            independent_field_mappings,
+            dependent_field_mappings,
+        ) = self.build_incoming_field_path_maps(group_dependent_fields)
+
+        if group_dependent_fields:
+            output[FIDESOPS_GROUPED_INPUTS] = []
```

Comment:
It feels brittle to use a whole arg just to initialise this var. This would imply that `dependent_field_mappings[collection_address]` referenced on line 229 below will always be Falsy if the `group_dependent_fields` var is `False`. Is it safer / feasible to initialise this var unconditionally?

Reply:
My thinking was that I didn't want to clutter the output for the `group_dependent_fields=False` case - this method is called twice, once to process incoming data into a collection, and the other to process outputs from a collection that are stored in the cache.   Also, the implication you mention above is true, if group_dependent_fields is False, it causes the built dependent_field_mappings for all collection addresses to always be empty so this seems safe to me. 

However, I agree we can add this key in both cases. It is removed in downstream processing because it's not an actual field on a collection, so it won't persist through to the results we send to the user.